repository/applicationRepository.ts

```
@@ -0,0 +1,48 @@
+import { Application } from "../models";
+import { AppParams } from "../utils/types";
+
+export class ApplicationRepository {
+    async findAll() {
+        try {
+            return await Application.find({
+                relations: ["company","participants"]
+            });
+        } catch (err) {
+            console.log(err);
+        }
+    }
+    async findById(appId: number) {
+        try {
+            return await Application.findOne({
+                where: {
+                    id: appId
+                }
+            });
+        } catch (err) {
+            console.log(err);
```

Throw errors everyone, no need for silent errors.